pepper-apis/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -46,7 +47,9 @@
      */
     private static final int PASSWORD_ROTATION_MAX_RETRIES = 2;
 
-    private HikariDataSource dataSource;
+    public static final String COULD_NOT_GET_CONNECTION = "Could not get a connection from the pool";
+
+    private HikariPool dataSource;
```

Comment:
I went with the `HikariPool` here instead of `HikariDataSource` because it opens up some interesting ability for testing and rotation, like zorching connections in the pool, or enabling suspension of the pool for automated password rotation (some other day).

Reply:
[EDIT]: Nope, gotta be `HikariDataSource`.  Otherwise the pool connection limit tests fail.